specifications/ReadersWriters/ReadersWriters.tla

```diff
@@ -0,0 +1,122 @@
+-------------------------- MODULE ReadersWriters --------------------------
+
+EXTENDS FiniteSets, Naturals, Sequences
+
+CONSTANT NumActors
+
+VARIABLES
+    readers, \* set of processes currently reading
+    writers, \* set of processes currently writing
+    waiting  \* queue of processes waiting to access the resource
+
+vars == <<readers, writers, waiting>>
+
+Actors == 1..NumActors
```

Comment:
Why not make Actors a constant parameter of the module instead of requiring that set to be an interval of integers? Then you can make Actors a set of model values, say {alice, bob, charlie}, in the model.

Reply:
I personally think it's better this way because you can easily change how many actors are in the model. I can make this change if you insist.